client/src/pages/League.tsx

```diff
@@ -1,18 +1,119 @@
+import { LeagueSelect } from '../components/league_select/LeagueSelect';
+import { LeagueStanding } from '../components/league_standing/LeagueStanding';
+import { LeagueSchedule } from '../components/league_schedule/LeagueSchedule';
 
 interface LeagueProps {
   className?: string;
 }
 
+enum TabType {
+  STANDING = 'STANDING',
+  SCHEDULE_SCORE = 'SCHEDULE_SCORE',
+}
+
 /**
  * League Page.
  * NOTE: This  might be broken into submenu per season.
  */
-const UnstyledLeague: FunctionComponent<LeagueProps> = () => {
-  return <Box>League page</Box>;
+const UnstyledLeague: FunctionComponent<LeagueProps> = ({ className }) => {
+  const { viewer } = useContext(ViewerContext);
+  const [teamTabType, setTeamTabType] = useState<TabType>(TabType.STANDING);
+  const { id } = useParams<{ id: string }>();
+
+  const leagueActive = useMemo(
+    () =>
+      find(
+        viewer.leagueActive,
+        (league) => league.id === parseInt(id, 10)
+      ), [id]
+  );
+
+  if (!leagueActive || !viewer.leagueTeamGroupAge || !viewer.matchesByAge) return <div>Loading...</div>
+
+  return (
+    <Box className={className}>
+      <Box
+        className="media-banner-container"
+        display="flex"
+        alignItems="center"
+      >
+        <Typography variant="h3" className="media-banner-text">
+          {leagueActive?.name} {leagueActive.year}
+        </Typography>
+      </Box>
+
+      <Container>
+        <Box
+          className="leagueSelect-box"
+          display="flex"
+          justifyContent="start"
+          mt={6}
+        >
+          <LeagueSelect
+            mt={7}
+            title="STANDING"
+            selected={teamTabType === TabType.STANDING}
+            onClick={() => setTeamTabType(TabType.STANDING)}
+          />
+
+          <LeagueSelect
+            mt={7}
+            title="SCHEDULE & SCORE"
+            selected={teamTabType === TabType.SCHEDULE_SCORE}
+            onClick={() => setTeamTabType(TabType.SCHEDULE_SCORE)}
+          />
+        </Box>
+
+        {teamTabType === TabType.STANDING && (
+          <LeagueStanding
+            className="standing-table"
+            teams={viewer.leagueTeamGroupAge[leagueActive.ageType]}
+          />
+        )}
+
+        {teamTabType === TabType.SCHEDULE_SCORE && (
+          <LeagueSchedule
+            className="schedule-table"
+            matches={viewer.matchesByAge[leagueActive.ageType]}
+          />
+        )}
+      </Container>
+    </Box>
+  );
 };
 
-export const League = withTheme(styled(UnstyledLeague)``);
+export const League = withTheme(styled(UnstyledLeague)`
+  .media-banner-container {
+    background-image: url(${AboutBanner});
+    min-width: 100px; /*or 70%, or what you want*/
+    height: 284px; /*or 70%, or what you want*/
+    background-size: 100% 100%;
+
+    .media-banner-text {
+      font-weight: 700;
+      color: white;
+      margin-left: 7rem;
+    }
+  }
+
+  .standing-table {
+    min-width: 1193px;
```

Comment:
This is gonna break if user changes window size.

Reply:
I think I dont need `min-width`. Fixed!